src/pages/library/dataBrowser-utils.js

```diff
@@ -38,10 +67,14 @@ const normalizeSnapshot = snapshot => {
     _.uniqBy(_.toLower)
   )(snapshot['prov:wasGeneratedBy'])
 
-  const dataReleasePolicy = _.flow(
-    _.replace('TerraCore:', ''),
-    _.startCase
-  )(snapshot['TerraDCAT_ap:hasDataUsePermission'])
+  const dataReleasePolicy = _.getOr(
+    {
+      ...snapshotReleasePolicies.releasepolicy_other,
+      desc: _.flow(
+        _.replace('TerraCore:', ''),
+        _.startCase
+      )(snapshot['TerraDCAT_ap:hasDataUsePermission'])
+    }, snapshot['TerraDCAT_ap:hasDataUsePermission'], snapshotReleasePolicies)
```

Comment:
How about something like this. We can then reference the policy using the name with the `TerraCore` prefix

```suggestion
  const dataReleasePolicy = _.has(snapshot['TerraDCAT_ap:hasDataUsePermission'], snapshotReleasePolicies) ?
    { policy: snapshot['TerraDCAT_ap:hasDataUsePermission'], ...snapshotReleasePolicies[snapshot['TerraDCAT_ap:hasDataUsePermission']] } :
    { policy: 'releasepolicyOther', ...snapshotReleasePolicies.releasepolicyOther }
```